Documentation/GettingStartedWithTheMRTK.md

```diff
@@ -116,10 +209,10 @@ Which contains the following:
 * **MixedRealityPlayspace** - The parent object for the headset, which ensures the headset / controllers and other required systems are managed correctly in the scene.
 * The Main Camera is moved as a child to the Playspace - Which allows the playspace to manage the camera in conjunction with the SDKs
 
->[!NOTE] 
+>[!NOTE]
 > While working in your scene, **DO NOT move the Main Camera** (or the **MixedRealityPlayspace**) from the scene origin (0,0,0).  This is controlled by the MRTK and the active SDK. If you need to move the players start point, then **move the scene content and NOT the camera**!
 
-6. Hit play and test out hand simulation by pressing spacebar.
+6. Hit play and test out hand simulation by pressing the **space bar**.
```

Comment:
This is making me question my own reality. What is it?!? spacebar or space bar?!

Reply:
it is referenced both ways in the doc. i just picked one. i can update all to be one word